src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,16 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+        double b = 1+((Math.pow(z,2)))/(3+(Math.pow(z,2)/5));
```

Comment:
It most case it's better to extract duplicated calculations into variable. So, please extract `Math.pow(z, 2)`.